rest/__init__.py

```diff
@@ -1,45 +1,44 @@
+from __future__ import absolute_import
 import flask
 import unicodecsv as csv
 
-from exceptions import ValueError
-
 from flask import Response
 from functools import wraps
-from StringIO import StringIO
+from io import BytesIO
```

Comment:
shouldnt we use six.moves here to make it backwards compat?

Reply:
yeah good catch i changed this to use six.BytesIO